infinit/content.md

```diff
@@ -0,0 +1,45 @@
+# What is Infinit?
+
+Infinit’s storage platform transforms commodity servers into multiple flexible, scalable, secure and fault-tolerant storage logics tailored to your applications.
+
+%%LOGO%%
+
+# How to use this image?
+
+The following commands show Docker-specific tips for a pleasant integration between Infinit and Docker. For the Infinit-specific documentation, consult [Infinit's website](https://infinit.sh/get-started).
+
+## Basics
+
+	docker run --rm infinit
+
+### Sharing configuration folder
+
+Infinit stores its configuration under `/root/.local/share/infinit`. If you want to share this folder with your host, you can mount your host's configuration folder as a volume in your container.
+
+	docker run -v ~/.local/share/infinit:/root/.local/share/infinit --rm infinit
+
+*N.B. This might affect permissions of the mounted folder.*
+
+### Using your user
+
+By default, Infinit uses the current user, `root` when running on a container. If you want to use your local user name, you can set the INFINIT_USER environment variable.
+
+	docker run -e INFINIT_USER=$USER -v ~/.local/share/infinit:/root/.local/share/infinit --rm infinit
+
+### Accessing storage logics
+
+For now, the only logic available is a distributed file system.
+
+#### File system
+
+On UNIX-based systems, Infinit uses `FUSE` to provide mount points. Hence, for UNIX-based containers, you need `FUSE` installed on your host (see Infinit's [get-started page](https://infinit.sh/get-started#installation)) and run Docker with specific options (`--cap-add SYS_ADMIN --device /dev/fuse`).
+
+	docker run --cap-add SYS_ADMIN --device /dev/fuse -e INFINIT_USER=$USER -v ~/.local/share/infinit:/root/.local/share/infinit --rm infinit volume mount --name <VOLUME> --mountpoint <DIRECTORY>
```

Comment:
I guess what I meant was making it a volume plugin on Docker and did not realize that plugins didn't work on swarm... until this next release!

So, it looks like volume plugins will work on swarm:

- PR: https://github.com/moby/moby/pull/33575
- merge commit https://github.com/moby/moby/commit/05c7c311390911daebcf5d9519dee813fc02a887
- commit in docker-ce: https://github.com/docker/docker-ce/commit/0887a18a3a2ea38cc4bf89e8f8e333be7288fe86

So, 17.07.0-ce-rc1 and above will work for running this as a volume plugin on a swarm. And I think an example or link to getting it to work as a plugin on a single docker would help users immensely.

Thanks!